apps/test/unit/lib/kits/maker/ui/Unit6ValidationStepTest.js

```diff
@@ -60,10 +61,10 @@ describe('Unit6ValidationStep', () => {
       <Unit6ValidationStep
         {...defaultProps}
         stepStatus={Status.SUCCEEDED}
-        initialChoice="yes1819"
+        initialChoice={Unit6Intention.YES_18_19}
       />
     );
-    assert.equal(wrapper.find('input [value="yes1819"]').props().checked, true);
+    assert.equal(wrapper.find(`input [value="${Unit6Intention.YES_18_19}"]`).props().checked, true);
```

Comment:
Is the ` intentional?

Reply:
Yep!  It's an ES6 [Template Literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) that allows us to do the variable interpolation `${}` within the string.